.gitignore

```diff
@@ -161,7 +161,8 @@ src/Hl7.Fhir.Model/EwoutFhirNetApi.snk
 *.snk
```

Comment:
Is this coming in from another fix?

Reply:
The newest version of Visual Studio places a map called ``.vs/`` in the root folder. This has to be ignored for our project